src/Slot-Core/ClassBinding.class.st

```diff
@@ -0,0 +1,8 @@
+"
```

Comment:
Why a new class? If there is no different behavior so long we can use LiteralVariable. 
Or better where are the places we have to refactor to take advantage of adding the new class.